sdk/python/feast/feature_view.py

```diff
@@ -241,3 +219,35 @@ def most_recent_end_time(self) -> Optional[datetime]:
         if len(self.materialization_intervals) == 0:
             return None
         return max([interval[1] for interval in self.materialization_intervals])
+
+    def infer_features_from_input_source(self):
+        if not self.features:
```

Comment:
Same comment as the datasource, should we assert that the features here are actually present in the input schema? or is that already happening elsewhere in the code?

Reply:
refer to reply to comment in datasource. I don't believe it's happening right now anywhere.

Reply:
Sounds good